_posts/2014-03-04-contacts.html

```diff
@@ -109,9 +109,16 @@ <h3 id="shortcut-methods">Shortcut Methods</h3>
     <tr>
       <td><span class="put-text">PUT</span></td>
       <td>
-        <code class="bluetext">contacts/{id}/unassign_tag/{tag_id}.format</code>
+        <code class="bluetext">contacts/{id}/split.format</code>
       </td>
-      <td>Remove a tag from a contact by name or ID</td>
+      <td>Split contact from a company. Send the field <code class="bluetext">company_name</code> in the body with the new company name.</td>
+    </tr>
+    <tr>
+      <td><span class="put-text">PUT</span></td>
+      <td>
+        <code class="bluetext">contacts/{id}/close_sales_cycle.format</code>
+      </td>
+      <td>Close the sales cycle for the contact. It's possible send a comment with the field <code class="bluetext">comment</code>.</td>
```

Comment:
@elanovasconcelos I would change the second sentence to:
"Add an optional closing comment with the <code class="bluetext">comment</code> field in the request body."